src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -80,6 +80,9 @@ private function loadXml(\DOMDocument $xml, string $path, \DOMNode $root = null)
         // parameters
         $this->parseParameters($xml, $path, $root);
 
+        // deprecated parameters
+        $this->parseDeprecatedParameters($xml, $root);
```

Comment:
I think deprecated parameters should be declared in DI extensions only. As such, I'd suggest removing all the code related to loaders/configurators

Reply:
Done and description of the PR updated.